app/controllers/posts_controller.rb

```diff
@@ -1,18 +1,22 @@
 class PostsController < ApplicationController
   def index
     # Catch the user which 'author_id' is the same as the parameter 'user_id' on the URL
-    user = User.find(params[:user_id])
+    @user = User.find(params[:user_id])
 
-    # catch all the posts associates to this user
-    @posts = user.posts
+    # Catch all the posts associates to this user and paginate them
+    @posts = @user.posts.page(params[:page]).per(3)
+
+    # @posts = @user.posts.paginate(page: params[:page], per_page: 4)
```

Comment:
[OPTIONAL] Please remove commented code from your codebase because it makes your codebase messy if you want to check previous code then check GitHub history :slightly_smiling_face: